### DISCUSSION TITLE
Change the default language

### DISCUSSION THREAD

**Discussion by Miftah96** [nuxt-modules/i18n#1354] [Q&A]
I installed the i18n package for switching languages. Then I set en (default) and id. So I want one of the pages (eg about) with the id language. The problem is I don't know the config? I use nuxt js

I tried the configuration, but it doesn't work :
`i18n: {`
    `      locales: [ 'en', 'id'],`
    `defaultLocale: 'en',`
    `parsePages: false,`
    `pages: {`
     ` paths: {`
      `  "software-house" : {`
       `   id: '/software-house',`
        `}`
      `},`
    `},`
    `vueI18n: {`
      `fallbackLocale: 'en',`
      `messages: {`
        `'en' : require('./locales/en.json'),`
        `'id' : require('./locales/id.json')`
      `},`
    `},`
  `},`

**Comment by rchl** [comment]
Just read https://i18n.nuxtjs.org/custom-paths#modules-configuration carefully and follow the structure that's documented. You seem to have an extra `paths` property in your config.

**Reply by rchl** [reply]
BTW. The code blocks should be surrounded by three backticks for readability -
\```
code
\```

**Reply by Miftah96** [reply]
Ok, thanks for the critique
